global.json

```diff
@@ -1,8 +0,0 @@
-{
-  "projects": [
-    "src"
```

Comment:
Unrelated to this PR, is there an equivalent replacement for the global.json hack now to compile a custom repo?